src/video_core/renderer_opengl/gl_rasterizer.cpp

```diff
@@ -1261,10 +1266,30 @@ void RasterizerOpenGL::SyncStencilTest() {
 
 void RasterizerOpenGL::SyncDepthTest() {
     const auto& regs = Pica::g_state.regs;
-    state.depth.test_enabled = regs.output_merger.depth_test_enable  == 1 ||
-                               regs.output_merger.depth_write_enable == 1;
-    state.depth.test_func = regs.output_merger.depth_test_enable == 1 ?
-                            PicaToGL::CompareFunc(regs.output_merger.depth_test_func) : GL_ALWAYS;
+
+    // Enable depth test so depth writes can still occur
+    state.depth.test_enabled = GL_TRUE;
```

Comment:
Shouldn't this still depend on `regs.output_merger.depth_write_enable` somehow?!